index.js

```diff
@@ -127,25 +145,23 @@ const drawCommand = (message) => {
   });
 };
 
-/*
-const inventoryCommand = () => {
-
-if user draws a card
-store name of card to firebase under the key cardName
-when the same user draws a second card, store the name of the second card under the previous draw within cardName
-continue to do this to all additional cards
-
-user{
-  userID{
-    lastDrawTime: xxxx
-    username: ""
-    cardName: "Blue Eyes White Dragon", "Red Eyes Black Dragon"
-  }
-}
-
+const inventoryCommand = (message) => {
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    const userIdentification = message.author.id;
+    const userInventoryAccess = userData[userIdentification].inventory;
+    let cardIDs = Object.keys(userInventoryAccess);
```

Comment:
this can also be a `const` (but we're going to get rid of this in a sec, too, so dont change this yet